packages/dev/gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -743,8 +743,10 @@ export class AdvancedDynamicTexture extends DynamicTexture {
      * @internal
      */
     public _changeCursor(cursor: string) {
-        if (this._rootElement) {
-            this._rootElement.style.cursor = cursor;
+        const scene = this.getScene();
+        if (this._rootElement && scene) {
+            scene._lastUsedCursor = this._rootElement.style.cursor;
+            this._rootElement.style.cursor = cursor !== "" ? cursor : scene._lastUsedCursor;

```

Comment:
you can use the scene's _setCursor function (public internal, should be available), and stay consistent in how we change cursor cross-packages

Reply:
Changed